### ISSUE TITLE
Work more closely with other great projects WCAG EM

### ISSUE THREAD

**Issue by mgifford** [wet-boew/wet-boew-wpss#115]
There are some great initiatives like https://github.com/w3c/wcag-em-report-tool
which can be fed by https://github.com/dequelabs/axe-reporter-earl
which every developer in the GC should be using as part of their CI as part of https://github.com/dequelabs/axe-core/
and which every content person should have access to Through Accessibility Insights which should also be able to be extended to support this standard publishing tool https://github.com/microsoft/accessibility-insights-web/issues/653

Are we not missing an opportunity to build with others and see that we have tools that make accessibility easier?
